scripts/ci/_utils.sh

```diff
@@ -39,10 +39,15 @@ function check_verbose_setup {
 # In case "VERBOSE" is set to "true" (--verbose flag in Breeze) all docker commands run will be
 # printed before execution
 function verbose_docker {
-    if [[ ${VERBOSE:="false"} == "true" ]]; then
+    if [[ ${VERBOSE_COMMANDS:=} == "true" ]]; then
```

Comment:
```suggestion
    if [[ ${VERBOSE} == "true" && ${VERBOSE_COMMAND} != "true" ]]; then
         # do not print echo if VERSBOSE_COMMAND is set (set -x does it already)
         echo "docker" "${@}"
     fi
     docker "${@}"
```

Reply:
Oh yes, can be simplified now. I originally had some branches doing `> /dev/null` but have since removed that.

Reply:
Fixed by https://github.com/apache/airflow/pull/9105/commits/ee5cb4c59d262bd2817d1bda56a3b2c0c31d6afe